flowapi/app/stream_results.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import ujson as json
+from quart import current_app
+
+
+async def stream_result_as_json(sql_query, headers={}, result_name="query_result"):
+    """
+    Generate a JSON representation of a query result.
+    Parameters
+    ----------
+    sql_query : str
+        SQL query to stream output of
+    headers : dict
```

Comment:
Why the name "headers" here? From the implementation it seems that these are included in the output before the actual streaming output, but this isn't apparent from the description. Can we either change the parameter name or explain this a bit more?